schemas/applications/schema/hardware.schema.json

```diff
@@ -27,7 +27,7 @@
           ]
         },
         "tf_prefix": {
-            "description": "The TF prefix to use for the URDF",
+            "description": "An optional TF prefix to apply to all of the links in the URDF to distinguish multiple instances of the same robot",
             "type": "string",
             "default": "",
```

Comment:
@eeberhard here I think an empty string could make sense, but perhaps we should also add some info on what happens if this stays empty.

perhaps a short:

> An optional TF prefix to apply to all of the links in the URDF to distinguish multiple instances of the same robot. If left empty, no changes will be made to the original URDF.